app/controllers/idv/phone_controller.rb

```diff
@@ -200,18 +200,22 @@ def failure_url(reason)
     end
 
     def async_state_done(async_state)
-      form_result = step.async_state_done(async_state)
+      results = step.async_state_done(async_state)
+      form_result = results[:final_result]
+      alternate_result = results[:alternate_result]
 
       analytics.idv_phone_confirmation_vendor_submitted(
         **form_result.to_h.merge(
           pii_like_keypaths: [
             [:errors, :phone],
-            [:context, :stages, :address],
+            [:context, :stages, :address], # does this do anything?
```

Comment:
should we address this in this ticket or come back to it with a later one?

Reply:
[Ticket](https://cm-jira.usa.gov/browse/LG-16989)